apps/src/lab2/views/components/Instructions/ResourcePanel/index.tsx

```diff
@@ -338,7 +339,13 @@ const ResourcePanel: React.FC<ResourcePanelProps> = ({
           id={currentTab}
           headerContent={tabInfo[currentTab].title}
           headerClassName={headerClassName}
-          rightHeaderContent={rightHeaderContent}
+          rightHeaderContent={
+            tabInfo[currentTab].title === commonI18n.aiTutor() ? (
```

Comment:
Can we key this off of the `Tabs` enum instead of comparing translated strings? `currentTab === Tabs.AiTutor`?